babel.config.js

```diff
@@ -0,0 +1,11 @@
+const { createConfig } = require('@edx/frontend-build');
+
+module.exports = createConfig('babel-preserve-modules', {
+  env: {
+    test: {
+      plugins: [
+        'rewire',
```

Comment:
What is this and do we really need it?

Reply:
It allows us to modify local module variables during testing.

Reply:
`getJwtToken.__Rewire__('httpClient', accessTokenAxios); // eslint-disable-line no-underscore-dangle`